src/storage/btree.c

```diff
@@ -14695,6 +14875,7 @@ btree_find_lower_bound_leaf (THREAD_ENTRY * thread_p, BTREE_SCAN * bts, BTREE_ST
       goto exit_on_error;
     }
 
+  COMMON_PREFIX_PAGE_SIZE_RESET (bts);
```

Comment:
Is there any reason to do it here?
Is initialization of common prefix information necessary to find the lower bound leaf?

Reply:
When bts->C_page changes, COMMON_PREFIX_PAGE_SIZE_RESET() is called to discard the current information.

Reply:
I understood that COMMON_PREFIX_PAGE_SIZE_RESET had to be called because bts->C_page was changed.